### PR TITLE
Groundworks for adding documentation

### PR DESCRIPTION
**Pull request by iharthi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,38 @@
...
+* [view](packages/view/README.md) - [Redux-Saga](https://github.com/redux-saga/redux-saga) runners used by SPA template
+* [view-manager](packages/view-manager/README.md) - view helpers used by SPA template 
+
+
+## Running the example app
+
+To run the example web application, install the dependencies and build the packages first:
+
+```
+yarn install
```
Comment (metsavaht) [inline README.md]: Initial commands should be
```
yarn install
yarn bootstrap
```

Contribution guide has some of the info as well, but main readme would be found more easily

Comment (iharthi) [reply README.md]: Added a link to CONTRIBUTING.md, fixed bootstrap commands list.

**Comment by metsavaht** [comment]
Please squash commits. Otherwise LGTM

**Comment by iharthi** [comment]
Squashed.

**Pull request merged**